### PR TITLE
command line interface

### PR DESCRIPTION
**Pull request by brisvag**
This would add `click` and `ipython` to the dependencies, but it makes interacting with star file super easy.

Replace the usual `less thing.star` with `starfile thing.star`, and you get dropped in an ipython console, ready to go and handle the star file with the power of pandas :)

### CODE REVIEW

**Comment by alisterburt** [comment]
this is really cool! love it - could you make the extra dependencies optional in `setup.cfg` under an 'interactive' header?

That way we could `pip install starfile[interactive]` to have this functionality but avoid the extra deps

example here 
https://github.com/napari/napari/blob/6ef2e2787341283af0f2659189202e2614a025e9/setup.cfg#L75-L117

**Comment by brisvag** [comment]
Sure! But I think `cli` is a better identifier :P

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+python_requires = >= 3.6
+include_package_data = True
+install_requires =
+    pandas>=1.0.5
+
+
+[options.extras_require]
+cli =
+    click
+    IPython
```
Comment (alisterburt) [inline setup.cfg:35]:
```suggestion
    click
    IPython
    matplotlib
    seaborn
```

File: starfile/__main__.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+try:
+    from IPython.terminal.embed import InteractiveShellEmbed
+    import click
```
Comment (alisterburt) [inline starfile/__main__.py:3]:
```suggestion
    import click
    import matplotlib.pyplot as plt
    import seaborn as sns
```

**Review by alisterburt — APPROVED** [review]
Thanks @brisvag !!

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
```
Comment (alisterburt) [inline setup.cfg:35]:
```suggestion
    click
    IPython
    matplotlib
    seaborn
```

File: setup.cfg [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    matplotlib
+    seaborn
+    matplotlib
+    seaborn
```
Comment (alisterburt) [inline setup.cfg:39]:
```suggestion
```

**Review by alisterburt — APPROVED** [review]

**Pull request merged**